lib/editor/util/map.js

```diff
@@ -775,7 +775,7 @@ export function stopToPatternStop (
       continuousDropOff: 1,
       continuousPickup: 1,
       dropOffBookingRuleId: null,
-      dropOffType: 0,
+      dropOffType: 3,
```

Comment:
No. We decided that if you need to change this you should do so via booking rules, as you otherwise have two ways of changing the same thing.

We can always change this in the future but as there is no real-world use of this yet we thought doing this would set a good standard for newly created data.